site/src/components/css-display/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./Css_table";
+export * from "./Accordian_view";
```

Comment:
most of the other components in the site are using PascalCasing for the filenames. For example AccordionView.tsx